node_modules/.gitignore

```diff
@@ -136,9 +136,6 @@
 !/mute-stream
 !/negotiator
 !/node-gyp
-!/node-gyp/node_modules/
-/node-gyp/node_modules/*
-!/node-gyp/node_modules/proc-log
```

Comment:
I pulled down these changes, and confirmed this is being removed because it's now dedupped.

```
├─┬ node-gyp@10.2.0
│ └── proc-log@4.2.0 deduped
```

Reply:
Yeah that was my bad, I didn't realize they'd made the PR into the cli itself.  Not my best day.